src/OrchardCore/OrchardCore.Abstractions/Localization/CultureScope.cs

```diff
@@ -6,6 +6,7 @@ namespace OrchardCore.Localization
     /// <summary>
     /// Represents a scope that you can change the current culture within.
     /// </summary>
+    /// <remarks>The scope disallow the current culture depends on local computer settings by default. For more information refer to https://github.com/OrchardCMS/OrchardCore/issues/11228</remarks>
```

Comment:
This is a really long line, add a line break in between. Same for CultureOptions.